mikeio1d/result_reader_writer/result_reader.py

```diff
@@ -81,6 +102,14 @@ def __init__(
 
         self.quantities = [quantity.Id for quantity in self.data.Quantities]
 
+        self.column_mode: ResultReader.ColumnMode = self.ColumnMode.ALL
```

Comment:
Lets use ColumnMode.QUERY as a default. Now the res1d.ipynb also gets broke with ColumnMode.ALL as a default.